v2/session/common-customizations/sessions/multiple-api-endpoints.mdx

```diff
@@ -112,7 +112,101 @@ Whilst the `cookieDomain` can start with a leading `.`, the value of the `apiDom
 For local development, you should not set the `cookieDomain` to an IP address based domain, or `.localhost` - browsers will reject these cookies. Instead, you should [alias `localhost` to a named domain and use that](https://superuser.com/questions/152146/how-to-alias-a-hostname-on-mac-osx). 
 :::
 
-## Step 2) Frontend config
+## Step 2) Set Older Cookie Domain in the Backend Config
```

Comment:
we should have some caution thing that says:
- The value set for olderCookieDomain must be kept for 1 year because the cookie lifetime of the access token on the frontend is 1 year.
- If you have changed the cookieDomain more than once within one year, to prevent a stuck state, switch to header based auth for all your clients (https://supertokens.com/docs/session/common-customizations/sessions/token-transfer-method#backend-configuration-optional). The important thing here is that you have to set the backend config to header even though that doc says it's optional. This is so that all clients are forced to use header based auth.

Reply:
This brings about one point of discussion. Let's say that an app has users in this situation where they already have cookies from domain `b.example.com` and `example.com`, and now the new cookieDomain is `a.b.example.com`. In this case, there would be three tokens coming to their APIs, and we would just clear one of the older ones instead of both the older ones. Would switching to header based auth (via backend header config) as mentioned in the above actually immediately log them out and allow them to login via header based auth (therefore solving this issue)?

Reply:
1. Done.

2. I am testing this. The logout -> signin -> getSession part is working fine after switched to header based auth. However, in the refresh api call, the accessToken is sent  instead of the refreshToken causing a logout. I am looking into this. I don't think this is related to cookie domain change. Could be some bad config on my end.

Reply:
when you say accessToken is sent instead of refresh token, you mean as an authorization bearer token? How did that happen? I mean even if it’s a config error, what config error caused this?